test/parallel/test-setproctitle.js

```diff
@@ -1,44 +1,73 @@
 'use strict';
-// Original test written by Jakub Lekstan <kuebzky@gmail.com>
 const common = require('../common');
 
+// Original test written by Jakub Lekstan <kuebzky@gmail.com>
+
 // FIXME add sunos support
 if (common.isSunOS) {
   return common.skip(`Unsupported platform [${process.platform}]`);
 }
 
 const assert = require('assert');
-const exec = require('child_process').exec;
+const { exec, spawn } = require('child_process');
 const path = require('path');
-
 // The title shouldn't be too long; libuv's uv_set_process_title() out of
 // security considerations no longer overwrites envp, only argv, so the
 // maximum title length is possibly quite short.
-let title = String(process.pid);
+const title = String(Date.now()).substr(-4, 4);
```

Comment:
Can you use a constant value then? I know this wasn’t the case before, but it’s not really a good idea to use random values in tests, where reproducibility matters (alternative: print the title to stderr, that should work as well)